ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -289,6 +288,25 @@ extension MSFListView {
     }
 }
 
+/// Generic UIView wrapper
+struct UIViewWrapper: UIViewRepresentable {
```

Comment:
This seems generic enough to be in a separate file to be leveraged by other controls.